### PR TITLE
Added update-home-page

### LINKED ISSUE #72: Update the homepage to include genres
Let's split up the artists from the genres using headings.
Make the title read: "Music of Madagascar"

Then include a link to the genres page, located at: `genres.md`

### PR DESCRIPTION
**Pull request by Tsikitsiky**
Fix #72 
Hey, 
This is our task, please have a look and comment.

Thank you,
Sarah and Sugi

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -26,4 +29,9 @@ Welcome to this list of Malagasy artists. If you want to know the famous artists
...
 - [X](x/index.md)
 - [Y](y/index.md)
-- [Z](z/index.md)
\ No newline at end of file
+- [Z](z/index.md)
+
+## Genres
+There are some genres wich are originated from each region in Madagascar, to know more click the link below.
+
+[Genre of Malagasy music](genres.md)
```
Comment (ndorfin) [inline index.md]: Small typo here, it should be `Genres`

File: index.md [hunk 2]
```diff
@@ -26,4 +29,9 @@ Welcome to this list of Malagasy artists. If you want to know the famous artists
 - [W](w/index.md)
 - [X](x/index.md)
 - [Y](y/index.md)
-- [Z](z/index.md)
\ No newline at end of file
+- [Z](z/index.md)
+
+## Genres
+There are some genres wich are originated from each region in Madagascar, to know more click the link below.
```
Comment (ndorfin) [inline index.md]: Small typo here too, it should read: `which originated`

**Review by ndorfin — CHANGES_REQUESTED** [review]
Two small fixes required:

File: index.md (hunk 2 above)
Comment (Tsikitsiky) [reply index.md]: We've sent the last change.

**Review by ndorfin — APPROVED** [review]
🚢 -it!

**Pull request merged**